src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -68,6 +68,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Exclude GAC, registry, output directory from search paths. -->
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
+
+    <AllowUnsafeBlocks Condition="'$(AllowUnsafeBlocks)'==''">false</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors Condition="'$(TreatWarningsAsErrors)'==''">false</TreatWarningsAsErrors>
+    <Prefer32Bit Condition="'$(Prefer32Bit)'==''">false</Prefer32Bit>
+    <SignAssembly Condition="'$(SignAssembly)'==''">false</SignAssembly>
+    <DelaySign Condition="'$(DelaySign)'!=''">false</DelaySign>
```

Comment:
The property pages are expecting a Boolean value and empty strings are throwing a wrench on things.

For your other question - I'll forward you the email threads with the discussion but basically yes they should be defaulted as it is the right thing from the project perspective.